Resources/Prototypes/Entities/Objects/Tools/tools.yml

```
@@ -543,6 +543,54 @@
     recipes:
       - RCD
 
+- type: entity
+  id: RPD
+  parent: RCD
+  name: RPD
+  description: A device used to rapidly pipe things.
+  components:
+  - type: Tag
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)